Documentation/ApiOverview/Ajax/Backend/Index.rst

```diff
@@ -150,7 +150,7 @@ compute our input and write the result into the console.
    require(['TYPO3/CMS/Core/Ajax/AjaxRequest'], function (AjaxRequest) {
      // Generate a random number between 1 and 32
      const randomNumber = Math.ceil(Math.random() * 32);
-     (new AjaxRequest(TYPO3.settings.ajaxUrls.example_dosomething).withQueryArguments({input: randomNumber}).get().then(async function (response) {
+     new AjaxRequest(TYPO3.settings.ajaxUrls.example_dosomething).withQueryArguments({input: randomNumber}).get().then(async function (response) {
```

Comment:
```suggestion
     new AjaxRequest(TYPO3.settings.ajaxUrls.example_dosomething)
       .withQueryArguments({input: randomNumber})
       .get()
       .then(async function (response) {
```

I think it would be useful to have a few linebreaks here for readability.